danuluma/store
# #161867569 add  logout functionality

#### What does this PR do?
add logout functionality on the frontend

#### Description of Task to be completed?
Implement logout functionality on the frontend

#### How should this be manually tested?
#### How should this be manually tested?
* Install the api as shown in https://github.com/danuluma/dannstore
* Clone or download this repository ```git clone https://github.com/danuluma/store.git``` https://github.com/danuluma/store/archive/ft-logout-161867569.zip
* Navigate to the project root then open UI folder
* Open index.html using your favorite browser(Chrome is recommended)
* Input the values the default name and password used during the api setup in the username and password fields
* Click the login button. You should be redirected to the products page.
* Now logout by clicking the logout button on the top right of the page



#### What are the relevant pivotal tracker stories?
[#161867569](https://www.pivotaltracker.com/story/show/161867569)
